src/NLU.DevOps.ModelPerformance/Utilities.cs

```diff
@@ -0,0 +1,119 @@
+       /// <param name="dividend"> The dividend in the division</param>
+       /// <param name="divisor"> The divisor in the division</param>
+       /// <returns>The division result</returns>
+        public static decimal Calculate(decimal dividend, decimal divisor)
+        {
+            return divisor != 0 ? decimal.Divide(dividend, divisor) : 0;
+        }
+
+        /// <summary>
+        /// Calculates the precision from a confusion matrix
+        /// </summary>
+        /// <param name="cm">confusion matrix metrics</param>
+        /// <returns>The precision result</returns>
+        public static decimal CalcPrecision(ConfusionMatrix cm)
+        {
+            return Calculate(cm.TruePositive, cm.TruePositive + cm.FalsePositive);
+        }
+
+        /// <summary>
+        ///  Calculates the recall from a confusion matrix
+        /// </summary>
+        /// <param name="cm"> confusin matrix metrics</param>
+        /// <returns> The recall result</returns>
+        public static decimal CalcRecall(ConfusionMatrix cm)
+        {
+            return Calculate(cm.TruePositive, cm.TruePositive + cm.FalseNegative);
+        }
+
+        /// <summary>
+        /// Calculates the f1 score from a confusion matrix
+        /// </summary>
+        /// <param name="cm"> confusion matrix metrics</param>
+        /// <returns> the f1 result</returns>
+        public static decimal CalcF1(ConfusionMatrix cm)
+        {
+            var precision = CalcPrecision(cm);
+            var recall = CalcRecall(cm);
+            var denominator = precision + recall;
+            return denominator != 0 ? 2 * decimal.Divide(precision * recall, denominator) : 0;
+        }
+
+        /// <summary>
+        /// Calculates Precision, Recall and F1 Score
+        /// </summary>
+        /// <param name="cm">confusion matrix metrics</param>
+        /// <returns>List that contains the results calculated</returns>
+        public static List<decimal> CalcMetrics(ConfusionMatrix cm)
+        {
+            List<decimal> metrics = new List<decimal>
+            {
+                CalcPrecision(cm),
+                CalcRecall(cm),
+                CalcF1(cm)
+            };
+            return metrics;
+        }
+
+        /// <summary>
+        /// Prints to the console the intents and entities performance results in a table
+        /// </summary>
+        /// <param name="statistics"> The computed data for intents and entities</param>
+        public static void PrintResults(NLUStatistics statistics)
+        {
+            Console.Out.WriteLine("== Intents results == ");
+            Console.Out.WriteLine("Intent          | Precision | Recall    | F1        |");
+            Console.Out.WriteLine("=====================================================");
+            Console.Out.Write(string.Format(CultureInfo.InvariantCulture, "{0,-15} |", "*"));
+
+            List<decimal> intentsTotalResults = Utilities.CalcMetrics(statistics.Intent);
```

Comment:
Can use `var` to minimize future diffs (e.g., if you trade `decimal` for `double`)